vxquery-server/src/main/resources/conf/local.xml

```diff
@@ -16,6 +16,7 @@
 -->
 <cluster xmlns="cluster">
     <name>local</name>
+    <indexDirectory>/home/menaka/index</indexDirectory>
```

Comment:
I think by default this should be the directory created in AbstractXQueryTests. Then you can create a bigger set of queries in src/test/resources/Queries/Indexing. It seems to me that these tests will fail currently on this branch, as they haven't been changed yet.